add-serve-target.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e -x
```

Comment:
Non-blocking: FYI, you can do this:
```suggestion
#!/bin/bash -ex
```

Reply:
Nice! Updated both this bash script and the preexisting one